envi/archs/thumb16/disasm.py

```diff
@@ -970,6 +977,8 @@ class Thumb16Disasm:
     _opclass = ThumbOpcode
 
     def disasm(self, bytez, offset, va, trackMode=True):
+        flags = 0
```

Comment:
true up VA and set flags.  instructions *always* actually start on even VA's, even thumb.